src/util/component-types.js

```diff
@@ -24,6 +25,7 @@ export function createClass(definition={}) {
 			selectors,
 			propName,
 		},
+		propTypes: _.assign({}, propTypes, _.mapValues(definition.components, () => React.PropTypes.any)),
```

Comment:
This change adds anything defined in `components` to the `propTypes` list for the component, to remove warnings when valid Child Components are being passed in as props.